docs/upgrade-to-v5.md

```diff
@@ -26,15 +25,15 @@ for Node.js and web platforms.
 
 ### Remove reliance on Node.js Buffer
 
-> **TL;DR**: Impacted APIs now return Uint8Array in web environments; Node.js environments unaffected
+> **TL;DR**: Impacted APIs now return `Uint8Array` in web environments; Node.js environments are unaffected
 
-For those that use the BSON library on Node.js, there is no change - the BSON APIs will still return and accept instances of Node.js Buffer. Since we no longer depend on the Buffer web shim for compatibility with browsers, in non-Node.js environments a Uint8Array will be returned instead.
+For those that use the BSON library on Node.js, there is no change - the BSON APIs will still return and accept instances of Node.js `Buffer`. Since we no longer depend on the `Buffer` web shim for compatibility with browsers, in non-Node.js environments a `Uint8Array` will be returned instead.
 
-This allows the BSON library to be better at platform independence while keeping its behavior consistent cross platform. The Buffer shim served the library well but brought in more than was necessary for the concerns of the code here.
+This allows the BSON library to be more platform independent while keeping its behavior consistent cross platform. The `Buffer` shim served the library well but brought in more than was necessary for the concerns of the code here.
```

Comment:
```suggestion
This allows the BSON library to be more platform independent while keeping its behavior consistent cross platform. 
```

I wrote this but I think this sentence doesn't add valuable information

Reply:
sgtm